.github/workflows/main.yml

```diff
@@ -22,19 +22,32 @@ jobs:
         run: |
           sudo mkdir /opt/local.old
           sudo mv /usr/local/* /opt/local.old
+      - name: Install Linux Dependencies
+        if: startsWith(matrix.os,'ubuntu')
+        run: >
+          sudo apt update &&
+          sudo apt install mtree-netbsd
+          gnustep libcurl4-gnutls-dev
```

Comment:
Is it correct for this to be split up across three lines?

Reply:
This should be fine. Instead of the usual `run: |`, I used `run: >`. This means that newlines are replaced with spaces (based off [here](https://stackoverflow.com/a/65808412)). Reviewing the logs, all the packages seem to be installed.